modular_zubbers/code/modules/research/techweb/_techweb.dm

```diff
@@ -0,0 +1,2 @@
+/datum/techweb
+	var/should_generate_points = TRUE
```

Comment:
Shouldn't this be just ``should_generate_points`` as the var is already defined elsewhere?